### PR TITLE
fix: cluster features on reearth/core

### PR DESCRIPTION
**Pull request by keiya01**
# Overview

### CODE REVIEW

File: src/components/organisms/Published/core/hooks.ts [hunk 1]
```diff
@@ -1,11 +1,11 @@
 import { mapValues } from "lodash-es";
 import { useState, useMemo, useEffect } from "react";
 
-import type { Block } from "@reearth/components/molecules/Visualizer";
+import type { Block, ClusterProperty } from "@reearth/components/molecules/Visualizer";
 import { config } from "@reearth/config";
 import type { InternalWidget, WidgetAlignSystem, WidgetAlignment } from "@reearth/core/Crust";
 import { convertLegacyLayer, type Layer, type LegacyLayer } from "@reearth/core/mantle";
-import type { Cluster } from "@reearth/core/Map";
+import { convertLegacyCluster } from "@reearth/core/mantle/compat/forward";
```
Comment (rot1024) [inline src/components/organisms/Published/core/hooks.ts:8]:
```suggestion
import { convertLegacyCluster } from "@reearth/core/mantle";
```

File: src/components/organisms/EarthEditor/core/CanvasArea/hooks.ts [hunk 2]
```diff
@@ -1,9 +1,11 @@
 import { useMemo, useEffect, useCallback } from "react";
 
+import { ClusterProperty } from "@reearth/components/molecules/Visualizer";
 import { config } from "@reearth/config";
 import type { Alignment, Location } from "@reearth/core/Crust";
 import { convertLegacyLayer } from "@reearth/core/mantle";
-import type { Cluster, Layer } from "@reearth/core/Map";
+import { convertLegacyCluster } from "@reearth/core/mantle/compat/forward";
```
Comment (rot1024) [inline src/components/organisms/EarthEditor/core/CanvasArea/hooks.ts:7]:
```suggestion
import { convertLegacyCluster } from "@reearth/core/mantle";
```

File: src/components/organisms/EarthEditor/core/CanvasArea/hooks.ts [hunk 3]
```diff
@@ -1,9 +1,11 @@
 import { useMemo, useEffect, useCallback } from "react";
 
+import { ClusterProperty } from "@reearth/components/molecules/Visualizer";
```
Comment (rot1024) [inline src/components/organisms/EarthEditor/core/CanvasArea/hooks.ts:3]: Can you copy ClusterProperty type here? We already know that Visualizer will be removed in the future, so let's clear up the dependencies beforehand.

**Review by rot1024 — CHANGES_REQUESTED** [review]

File: src/components/organisms/EarthEditor/core/CanvasArea/hooks.ts (hunk 3 above)
Comment (keiya01) [reply src/components/organisms/EarthEditor/core/CanvasArea/hooks.ts]: I fixed in this commit
[fix: import](https://github.com/reearth/reearth-web/pull/430/commits/cc4d7fc81fd4c2cac618c9caf5f51df7f42c9c0f)

**Review by rot1024 — APPROVED** [review]

**Pull request merged**